_overviews/psp/overview.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Overview
+topic: PSP
+layout: overview
+sectionid: overview
+redirect_from:
+  - /overview
+---
+
+## What is the Plugin Server Protocol?
+
+If you are not aware of the Language Server Protocol, please read [this overview]({{ "/overviews/lsp/overview" | prepend: site.baseurl }}) first
+<br>
+<br>
+
+The idea behind the *Plugin Server Protocol (PSP)* is to standardize the protocol between editors (clients) and plugins (servers); so that a single Plugin Server can be re-used in multiple development tools, and tools can support plugins with minimal effort.
+
+PSP is a win for both plugin providers and tool vendors!
+
+## How it works
+
+The Plugin Server Protocol serves as an extension of the existing Language Server Protocol, which standardized how language tooling communicated with editors. Much of the communication and methods are inherited from it, though with some changes:
+
+- New fields in the initialize handshake to subscribe to events that the plugin wishes to receive. A plugin will receive only events it has subscribed to.
```

Comment:
Probably need to modify what this line says when you remove `subscribedMethods`

Reply:
If you get rid of the line completely then it also isn't a list anymore, so will need some sentence reformulating. Though, I think it shouldn't be a list anyway due to only having two elements..

Reply:
Or is this referring to the usual `psp`/`dap`/etc parts of the document?

Reply:
This has been reworked to remove the list.